src/main/java/page_objects/BasePage.java

```diff
@@ -29,4 +35,29 @@ public void gotoBookTicketPage() {
     public void gotoContactPage() {
         BrowserHelper.getDriver().findElement(tabContact).click();
     }
+
+    public void gotoChangePasswordPage() {
+        BrowserHelper.getDriver().findElement(tabChangePassword).click();
+    }
+
+    public void gotoMyTicketPage() {
+        BrowserHelper.getDriver().findElement(tabMyTicket).click();
+    }
+
+    public boolean isMyTicketTabExist() {
```

Comment:
`exist` is a verb

Reply:
I changed Exist to Existent.

Reply:
incorrect. there 2 two mistakes here
1. is...Existent is not a correct sentence
2. inside the method you are checking isDisplayed() but you name your method is Existent...

Reply:
Thank you for your review. I've updated new code, please review it again.